_layouts/cv.liquid

```diff
@@ -54,12 +54,17 @@ layout: default
         {{ page.title }}
         {% if page.cv_pdf %}
           <a
-            href="{{ page.cv_pdf | prepend: 'assets/pdf/' | relative_url}}"
+            {% if page.cv_pdf contains 'http' %}
```

Comment:
Maybe it is best to search for `://` instead of `http`, since it is possible to have the http string, and also to support, for example, storing the cv file in a `ftp://`.

Reply:
Makes sense.